### PR TITLE
bug 2059292: Fix error when deploying sriov network operator

### PR DESCRIPTION
**Pull request by bn222**
https://bugzilla.redhat.com/show_bug.cgi?id=2059292

### CODE REVIEW

**Comment by zshi-redhat** [comment]
@bn222 We would need a 4.10 and master BZs (if not already exist) in verified state in order for this PR to be labeled with bugzilla/valid-bug.

File: manifests/4.9/supported-nic-ids_v1_configmap.yaml [hunk 1]
```diff
@@ -22,7 +22,6 @@ data:
   Nvidia_mlx5_ConnectX-5_Ex: 15b3 1019 101a
   Nvidia_mlx5_ConnectX-6: 15b3 101b 101c
   Nvidia_mlx5_ConnectX-6_Dx: 15b3 101d 101e
-  Nvidia_mlx5_ConnectX-6_Lx: 15b3 101f 101e
```
Comment (SchSeba) [inline manifests/4.9/supported-nic-ids_v1_configmap.yaml:25]: is there a reason why we remove this device?

**Comment by SchSeba** [comment]
@bn222 is the change exists in 4.10 I was able to find it.

we are also deploying the sriov operator as part of the CI and didn't encounter this issue.

File: manifests/4.9/supported-nic-ids_v1_configmap.yaml (hunk 1 above)
Comment (bn222) [reply manifests/4.9/supported-nic-ids_v1_configmap.yaml:25]: Same as here: https://github.com/openshift/sriov-network-operator/pull/639#discussion_r833046689

**Comment by bn222** [comment]
> @bn222 is the change exists in 4.10 I was able to find it.
> 
> we are also deploying the sriov operator as part of the CI and didn't encounter this issue.

That's a bit unexpected. On a clean 4.9 env using dev-scripts, I can not deploy the sriov network operator. Any ideas if the CI is not caching anything from previous runs?

**Comment by bn222** [comment]
Related bug has been closed with WORKSFORME

**Pull request closed without merging**